### PR TITLE
Individual Bluespace Jumping

### PR DESCRIPTION
**Pull request by ZephyrTFA**
## About The Pull Request

Adds the ability for ships to individually bluespace jump.

## Why It's Good For The Game

Because I said it is 🥂 

## Changelog
:cl:
add: Ships can now individually bluespace jump.
/:cl:

### CODE REVIEW

**Review by MarkSuckerberg — APPROVED** [review]
Looks good, should be testmerged and stress tested first though

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: ca83a63d9154e7bd47271dd9f5ce5d44eb19d7af
Pull Request: 3e74246da56b73aa3823370030ddea8e41408e7e
Server: b9679cc28f47ef4054511559ac3e5f35dd3100bb

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: ca83a63d9154e7bd47271dd9f5ce5d44eb19d7af
Pull Request: 3e74246da56b73aa3823370030ddea8e41408e7e
Server: 52f18b307da9ed9bafb2f173c5db9806fde13a70

**Comment by Azarak** [comment]
I saw two new players use this thinking it'd teleport their ship to a random location, I think a more blunt ooc-information warning would be good maybe?

File: code/modules/overmap/helm.dm [hunk 1]
```diff
@@ -168,6 +232,14 @@
 					S.current_autopilot_target = null
 					S.burn_engines()
 					return
+				if("bluespace_jump")
+					if(tgui_alert(usr, "WARNING: ONCE STARTED A BLUESPACE JUMP CANNOT BE CANCELLED. ARE YOU SURE?", "Jump Confirmation", list("Yes", "No")) != "Yes")
+						return
+					if(tgui_alert(usr, "ARE YOU SURE YOU WANT TO BLUESPACE JUMP?", "Jump Confirmation", list("Yes", "No")) != "Yes")
```
Comment (ZephyrTFA) [inline code/modules/overmap/helm.dm]:
```suggestion
					if(tgui_alert(usr, "ARE YOU SURE YOU WANT TO BLUESPACE JUMP? THIS DELETES YOUR ENTIRE SHIP!", "Jump Confirmation", list("Yes", "No")) != "Yes")
```
```suggestion
					if(tgui_alert(usr, "ARE YOU SURE YOU WANT TO BLUESPACE JUMP?", "Jump Confirmation", list("Yes", "No")) != "Yes")
```

**Pull request merged**